bugbug/tools/code_review.py

```diff
@@ -369,8 +369,50 @@ def get_all_inline_comments(
                 yield diff_id, comments
 
 
+class SwarmReviewData(ReviewData):
+    def __init__(self):
+        self.auth = {
+            "user": get_secret("SWARM_USER"),
+            "password": get_secret("SWARM_PASS"),
+            "port": get_secret("SWARM_PORT"),
+            "instance": get_secret("INSTANCE"),
```

Comment:
You may want to follow the same pater by adding the `SWARM` prefix to avoid mixing up vars :)
```suggestion
            "instance": get_secret("SWARM_INSTANCE"),
```